packages/terra-form/src/components/ChoiceField.jsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable jsx-a11y/label-has-for */
+
+import React, { PropTypes } from 'react';
+
+import Control from './Control';
+import Field from './Field';
+
+const propTypes = {
+  /**
+   * Choices to populate radio buttosn for
+   */
+  choices: PropTypes.array,
```

Comment:
+1 to that idea. I think that looks more clear than passing in choiceName and choiceValue as well.